### PR TITLE
1.2.samples

### PR DESCRIPTION
**Pull request by eivantsov**
This PR:

1. Replaces WildFly with Thorntail. Impacts tests
2. Replaces Fuse sample application. Impact tests
3. Changes some commands, mainly allocated more memory
4. Adds commands to stacks (rather than samples only. This way it is possible to create a workspace from dashboard and import a project in the IDE, and still have commands)

### CODE REVIEW

File: ide/codeready-ide-samples/src/main/resources/samples.json [hunk 1]
```diff
@@ -31,16 +31,24 @@
...
+        "commandLine": "export MAVEN_OPTS='-Xms200m'\nmvn -f ${current.project.path}/pom.xml clean install\ncp ${current.project.path}/target/*.war /opt/eap/standalone/deployments/ROOT.war\nexport JAVA_OPTS= && export JAVA_OPTS_APPEND=-Dsun.util.logging.disableCallerCheck=true && /opt/eap/bin/standalone.sh -b 0.0.0.0 --debug 8000",
         "attributes": {
           "previewUrl": "${server.eap}",
           "goal": "Run"
         }
       },
+      {
+        "name": "copy-war",
+        "type": "mvn",
+        "commandLine": "p ${current.project.path}/target/*.war /opt/eap/standalone/deployments/ROOT.war",
```
Comment (rhopp) [inline ide/codeready-ide-samples/src/main/resources/samples.json]: missing "c"?

Comment (eivantsov) [reply ide/codeready-ide-samples/src/main/resources/samples.json]: Thanks. Fixed

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-OKD

**Comment by SkorikSergey** [comment]
ci-test-OKD

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

**Comment by artaleks9** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test-OKD

**Comment by artaleks9** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test-OKD

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
Selenium tests for ```Thorntail``` and ```Red Hat Fuse``` stacks adapted.

**Pull request merged**